core/chains/evm/txmgr/reaper_test.go

```diff
@@ -54,10 +53,8 @@ func TestReaper_ReapEthTxes(t *testing.T) {
 	cltest.MustInsertConfirmedEthTxWithReceipt(t, borm, from, nonce, 5)
 
 	t.Run("skips if threshold=0", func(t *testing.T) {
-		config := new(mocks.ReaperConfig)
-		config.On("EvmFinalityDepth").Return(uint32(10))
+		config := mocks.NewReaperConfig(t)
 		config.On("EthTxReaperThreshold").Return(0 * time.Second)
-		config.On("EthTxReaperInterval").Return(1 * time.Hour)
```

Comment:
Curious what the reason is for removing these On's, are they not called any more?

Reply:
Oh I guess with new() it didn't care whether they were called, but now it does

Reply:
yeah, none of these were actually checked before